### PR TITLE
Enable container publication to GitHub Container Registry.

### PR DESCRIPTION
**Pull request by felddy**
#  #

## 🗣 Description ##

Adds the ability for the images generated in the actions workflows to be pushed to the [Github Container Registry](https://docs.github.com/en/packages/guides/about-github-container-registry).

Images are now pushed to both registries.

## 💭 Motivation and Context ##

DockerHub is increasingly hostile towards open projects and the teams that support them.
* team size limits
* image pull limits
* image retention limits

This is a hedge against DockerHub spontaneously disabling support for all free projects, and a stepping stone towards adding [ECR](https://aws.amazon.com/ecr/) support.

## 🧪 Testing ##

Tested locally and in CI.  And in a personal project. 

A package has already been generated and published to registries as a result of this branch: 
* https://github.com/orgs/cisagov/packages/container/package/example
* https://hub.docker.com/r/cisagov/example

All cisagov packages in a single location:
* https://github.com/orgs/cisagov/packages

## 📷 Screenshots (if appropriate) ##

kiwi

## ✅ Checklist ##

* [x] This PR has an informative and human-readable title.
* [x] Changes are limited to a single goal - _eschew scope creep!_
* [x] All relevant type-of-change labels have been added.
* [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
* [x] These code changes follow project standards.
* [x] All relevant repo and/or project documentation has been updated to reflect
      the changes in this PR.
* [x] All new and existing tests pass.

### CODE REVIEW

**Review by hillaryj — APPROVED** [review]
🥡

**Review by jsf9k — APPROVED** [review]

**Comment by felddy** [comment]
Waiting for the Krakening... pass it on.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -275,9 +279,10 @@ jobs:
   build-push-all:
     # Builds the final set of images for each of the platforms listed in
     # PLATFORMS environment variable.  These images are tagged with the Docker
-    # tags calculated in the "prepare" job and pushed to DockerHub.  The
-    # contents of README.md is pushed as the image's description.  This job is
-    # skipped when the triggering event is a pull request.
+    # tags calculated in the "prepare" job and pushed to DockerHub and the
+    # GitHub Container Registry.  The contents of README.md is pushed as the
```
Comment (mcdonnnj) [inline .github/workflows/build.yml]:
```suggestion
    # GitHub Container Registry.  The contents of README.md are pushed as the
```

**Review by mcdonnnj — APPROVED** [review]
Solid improvement 💪

**Review by dav3r — APPROVED** [review]
Looks solid to me. 👍 👍  🚫 🐳

**Pull request merged**